.ci/validation.sh

```diff
@@ -277,6 +277,7 @@ no-error-xwiki)
   echo "Checkout target sources ..."
   checkout_from "https://github.com/xwiki/xwiki-commons.git"
   cd .ci-temp/xwiki-commons
+  git checkout "02b8c776439f37757ccc8ea8b84ea68fba8c7ca0"
```

Comment:
```suggestion
  git checkout "02b8c776439f37757ccc8ea8b84ea68fb""a8c7ca0"
```

Reply:
No need to spend time on this, no-error-xwiki is still failing. The above suggestion should be enought to guide you on spellcheck workarounds for whatever stable commit we land on.

Reply:
> above suggestion should be enough to guide you on spellcheck for whatever stable commit we land on.

We found that  no-error-xwiki is failing despite the build success because of the commit reference : 

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  37.640 s
[INFO] Finished at: 2024-02-07T04:30:48Z
[INFO] ------------------------------------------------------------------------
Checkout target sources ...
Cloning into 'xwiki-commons'...
Warning: Permanently added the ECDSA host key for IP address '140.82.113.3' to the list of known hosts.
remote: Enumerating objects: 257721, done.        
remote: Counting objects: 100% (565/565), done.        
remote: Compressing objects: 100% (482/482), done.        
remote: Total 257721 (delta 272), reused 336 (delta 66), pack-reused 257156        
Receiving objects: 100% (257721/257721), 35.12 MiB | 32.63 MiB/s, done.
Resolving deltas: 100% (117783/117783), done.
fatal: reference is not a tree: 02b8c776439f37757ccc8ea8b84ea68fba8c7ca0
```

Could you please elaborate on what do you mean by a `Stable commit` ?

Reply:
Look at git blame in the area around this change, and look at previous changes that have been done when this has happened previously. We will need a commit in their default branch that reliably builds and makes our CI happy.

Reply:
> Look at git blame in the area around this change, and look at previous changes that have been done when this has happened previously. We will need a commit in their default branch that reliably builds and makes our CI happy.

I chose a stable commit and  `ci/circleci: no-error-xwiki`  seems to be "happy"   :)
All the other CI also are green.... Please suggest if any changes need to be done

Reply:
Please add a GitHub link to this commit to the PR description.

Reply:
> Please add a GitHub link to this commit/SHA in xwiki repo to the PR description.

Done.... Updated PR description